Sources/MulticastDelegate.swift

```diff
@@ -128,7 +128,8 @@ public func -=<T>(left: MulticastDelegate<T>, right: T) {
  *  - returns: The `MulticastDelegate` after all its delegates have been invoked
  */
 infix operator |> { associativity left precedence 130 }
-public func |><T>(left: MulticastDelegate<T>, @noescape right: (T) -> ()) {
+public func |><T>(left: MulticastDelegate<T>, @noescape right: (T) -> ()) -> MulticastDelegate<T> {
```

Comment:
This warning is erroneous and should be ignored... Swift requires whitespace here.